src/pages/components/image-with-caption.mdx

```diff
@@ -29,6 +30,16 @@ The Image with caption supports images at 1x1, 2x1, 4x3, and 16x9 aspect ratios.
 </Column>
 </Row>
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline (caption) | Short copy | Yes | 1 | XXS (40 / 55)|
```

Comment:
Do we need something in the way of hierarchy for our headings? Like h1, h2?

Reply:
Do you mean hierarchy in the way that we display the lines of the content model?

Reply:
No, I mean of the component or heading itself. This isn't the best example now that I think about it because the heading is the copy in this scenario.